Project.toml

```diff
@@ -4,22 +4,24 @@ authors = ["Jose Daniel Lara", "Alessandro Francesco Castelli", "Sourabh Dalvi"]
 version = "0.12.1"
 
 [deps]
+AppleAccelerate = "13e28ba4-7ad8-5781-acae-3021b1ed3924"
```

Comment:
There is no need to add this as a dependency

Reply:
It might be an oversight from previous code

Reply:
So should I remove this then? https://github.com/NREL-Sienna/PowerNetworkMatrices.jl/blob/8ce328926dc1e862a39a114a29fcea61a2c5f0b3/src/PowerNetworkMatrices.jl#L41